packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -56,12 +62,14 @@ class ButtonGroup extends React.Component {
   render() {
     const {
       children,
+      isBlock,
       onChange,
       selectedKeys,
       ...customProps
     } = this.props;
 
     const buttonGroupClassNames = cx(['button-group',
+      { block: isBlock },
```

Comment:
nitpick - want to refactor the css class to `is-block` to make it 1-1 with the prop?

Reply:
+1, for reference -- was set to match button which has the class "block", so on next uplift of button, we should add it to an issue to update it there as well so we have parity.